Makefile

```diff
@@ -50,7 +50,7 @@ build-bin-release:
 
 build: 
 	$(shell if [ ! -e $(BINARYDIR) ];then mkdir -p $(BINARYDIR); fi)
-	go build -o bin/multicluster-controlplane cmd/server/main.go 
+	go build -ldflags="-s -w" -o bin/multicluster-controlplane cmd/server/main.go 
```

Comment:
we need it in stolostron/multicluster-controlplane

Reply:
yes, we will